### PR TITLE
Allow to change dt locale

### PR DESCRIPTION
**Pull request by lenisko**
Allow to set locale for dates & times using config.
As `json` is not providing a way to set `undefined` we lose auto detection of locale.

Read more here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

### CODE REVIEW

File: src/data/default.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const data = require('../../static/locales/' + config.locale + '.json');
 data.title = config.title;
 data.header_title = config.headerTitle;
 data.locale = config.locale;
+data.dateTimeLocale = config.dateTimeLocale;
```
Comment (versx) [inline src/data/default.js:8]: Snake case please like the other mustache keys.
```suggestion
data.date_time_locale = config.dateTimeLocale;
```

File: src/views/index.mustache [hunk 2]
```diff
@@ -44,6 +44,7 @@
         const configPvp = {{{config_pvp_json}}};
 
         const locale = '{{{locale}}}';
+        const dateTimeLocale = '{{{dateTimeLocale}}}';
```
Comment (versx) [inline src/views/index.mustache:47]: Same here
```suggestion
        const dateTimeLocale = '{{{date_time_locale}}}';
```

**Review by versx — CHANGES_REQUESTED** [review]
Looks good

**Review by versx — APPROVED** [review]

**Review by TurtIeSocks — APPROVED** [review]
Nicely done

**Pull request merged**